client/modules/IDE/components/Editor.jsx

```diff
@@ -168,8 +167,9 @@ class Editor extends React.Component {
     });
 
     this._cm.on('keydown', (_cm, e) => {
-      // 9 === Tab
-      if (e.keyCode === 9 && e.shiftKey) {
+      // 70 === f
+      if ((e.metaKey || e.ctrlKey) && e.shiftKey && e.keyCode === 70) {
```

Comment:
i think it would also make sense to check the value of [`metaKey`](https://github.com/processing/p5.js-web-editor/blob/059b1710c74058bf5801df4d778cc9215bd5a87c/client/modules/IDE/components/Editor.jsx#L39), so that Ctrl + Shift + F doesn't work on Mac.

Reply:
Yes, I have added the check accordingly!